phpBB/includes/functions_content.php

```diff
@@ -327,18 +327,14 @@ function bump_topic_allowed($forum_id, $topic_bumped, $last_post_time, $topic_po
 */
 function get_context(string $text, array $words, int $length = 400): string
 {
-	if ($length <= 0)
-	{
-		return '...';
```

Comment:
Just return $text, is more simple i think

Reply:
Makes sense.